MFVNeutralino/plugins/VertexTracks.cc

```diff
@@ -665,16 +661,16 @@ bool MFVVertexTracks::filter(edm::Event& event, const edm::EventSetup& setup) {
     std::random_shuffle(all_tracks->begin(), all_tracks->end(), random_converter);
   }
 
+  TRandom3 *r3 = new TRandom3(); //Alec added
   for (size_t i = 0, ie = all_tracks->size(); i < ie; ++i) {
     const reco::TrackRef& tk = (*all_tracks)[i];
-    const auto rs = track_rescaler.scale(*tk);
+    const auto rs = (track_rescaler_which == 1) ? track_rescaler.scale(*tk, "")
+                  :  track_rescaler.scale(*tk);
```

Comment:
@abbywarden here and elsewhere we should either use the enum value for track_rescaler_which (in case it ever changes) or set the "" argument to be the default in some form, to avoid needing this at all.